### ISSUE TITLE
magit recipe missing a file

### ISSUE THREAD

**Issue by prsteele** [milkypostman/melpa#3487]
I believe the recipe for magit is missing the file ``magit-autorevert.el``. As of now the elisp command ``magit-status`` fails, complaining about that file missing. If I manually add that file from https://github.com/magit/magit everything works correctly. I believe the recipe should be

```(magit :fetcher github
       :repo "magit/magit"
       :files ("lisp/magit-utils.el"
               "lisp/magit-section.el"
               "lisp/magit-git.el"
               "lisp/magit-mode.el"
               "lisp/magit-process.el"
               "lisp/magit-core.el"
               "lisp/magit-diff.el"
               "lisp/magit-wip.el"
               "lisp/magit-apply.el"
               "lisp/magit-log.el"
               "lisp/magit.el"
               "lisp/magit-sequence.el"
               "lisp/magit-commit.el"
               "lisp/magit-remote.el"
               "lisp/magit-bisect.el"
               "lisp/magit-stash.el"
               "lisp/magit-blame.el"
               "lisp/magit-ediff.el"
               "lisp/magit-extras.el"
               "lisp/magit-autorevert.el"
               "lisp/git-rebase.el"
               "Documentation/magit.texi"
               "Documentation/AUTHORS.md"
	       "COPYING"))
```

**Comment by syohex** [comment]
Please see #3488

**Comment by purcell** [comment]
Fixed as part of #3488. New package has not yet built, though.
